SAPex/appsettings.json

```diff
@@ -11,10 +11,11 @@
   "AppSettings": {
     "Secret": "PDv7DrqznYL6nv7DrqzjnQYO9JxIsWdcjnQYL6nu0f",
     "ExpDate": 1,
-    "ExpDay": 7
+    "ExpMonth": 7
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(LocalDB)\\MSSQLLocalDB;Database=SAPExDb;Trusted_Connection=True;MultipleActiveResultSets=true;"
+    "DefaultConnection": "Server=127.0.0.1,1433;Database=SAPExDb;User Id=SA;Password=MyPass@word;"
+    //"DefaultConnection": "Server=(LocalDB)\\MSSQLLocalDB;Database=SAPExDb;Trusted_Connection=True;MultipleActiveResultSets=true;"
```

Comment:
Please comment on your personal connection string and uncomment the previous connection string.